opencog/atoms/atom_types/atom_types.script

```diff
@@ -438,6 +438,7 @@ TYPE_CO_INH_NODE <- TYPE_NODE
 DEFINED_TYPE_NODE <- TYPE_NODE
 
 // Used for type polymorphism.
+// Note that the empty type choice corresponds to the bottom type.
```

Comment:
It looks good to me @kasimebrahim , could you just add in the comment a link to that wiki page https://wiki.opencog.org/w/TypeChoice#Bottom_type that justifies why `TypeChoice` is equivalent to the bottom type. Thanks.